hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/replication/ContainerImporter.java

```diff
@@ -120,9 +123,15 @@ public void importContainer(long containerID, Path tarFilePath,
       try (InputStream input = Files.newInputStream(tarFilePath)) {
         Container container = controller.importContainer(
             containerData, input, packer);
-        // After container import is successful, increase used space for the volume
+        // After container import is successful, increase used space for the volume and schedule an OnDemand scan for it
         targetVolume.incrementUsedSpace(container.getContainerData().getBytesUsed());
         containerSet.addContainerByOverwriteMissingContainer(container);
+        Optional<Future<?>> scanFuture = OnDemandContainerDataScanner.scanContainer(container);
+        if (scanFuture.isPresent()) {
+          LOG.info("Scheduled on-demand scan for imported container {}", containerID);
+        } else {
+          LOG.debug("Skipped on-demand scan for imported container {}", containerID);
+        }
```

Comment:
- `scanContainer` should log whether it triggered a scan or not, so that callers do not need to repeat this logic
- add a parameter to indicate the reason ("import" in this case) to let `scanContainer` include it in the log